ts/commands/PackageCommands.ts

```diff
@@ -21,6 +21,14 @@ class PackageCommands {
         packageValidator.validate(location, require(location + '/package.json').id);
       });
 
+    (caporal as any)
+    .command('package create', 'Create a local pkg.zip')
+    .action((args, options, logger) => {
+      const location: string = path.resolve(process.cwd());
+      const config = configService.mergeConfig(options);
+      packageService.createArchive(location, config, logger);
```

Comment:
Location is undefined here.

Reply:
I can log it and it's defined for me.  It's the current working directory.

Reply:
yeah, there is no --location option, I suppose I could add one.